apps/judicial-system/backend/src/app/formatters/caseFilesRecordPdf.ts

```diff
@@ -0,0 +1,177 @@
+import { FormatMessage } from '@island.is/cms-translations'
+import {
+  capitalize,
+  caseTypes,
+  formatNationalId,
+} from '@island.is/judicial-system/formatters'
+
+import { caseFilesRecord } from '../messages'
+import { Defendant } from '../modules/defendant'
+import { Case } from '../modules/case'
+import { Alignment, PageLink, PdfDocument } from './pdf'
+
+function formatDefendant(defendant: Defendant) {
+  let nationalId = ''
+  if (defendant.noNationalId) {
```

Comment:
Isn't this a bit strange? You first check for noNationalId and then nationalId. How can nationalId be set if noNationalId is set?

Reply:
`nationalId` is also used for date of birth, so it can be set whether `noNationalId` is `true` or `false`. However, date of birth is optional.